ta_storage/base.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+from typing import Any
+
+from database.models.reports import Upload
+
+
+class TADriver(ABC):
+    @abstractmethod
+    def write_testruns(
+        self,
+        repo_id: int,
+        commit_id: str,
```

Comment:
do you think its reasonable to name this `commit_sha` rather? I know internally its often called `commitid`, but its too easy to confuse that with the `Commit.id` DB primary key.